src/status_im/ui/components/permissions.cljs

```diff
@@ -1,5 +1,5 @@
 (ns status-im.ui.components.permissions
```

Comment:
would be cool to move this namespace to react-native.permissions

Reply:
Sure thing 🫡

Reply:
Done